ldm/generate.py

```diff
@@ -567,7 +569,11 @@ def _load_model_from_config(self, config, weights):
 
         # this does the work
         c     = OmegaConf.load(config)
-        pl_sd = torch.load(weights, map_location='cpu')
+        with open(weights,'rb') as f:
+            weight_bytes = f.read()
```

Comment:
Reading weight_bytes is adding roughly 1.5s to every dream.py start up for me. Please move this off start up path and into _cached_sha256.